mmv1/products/firestore/Index.yaml

```diff
@@ -96,6 +96,16 @@ properties:
     values:
       - :COLLECTION
       - :COLLECTION_GROUP
+      - :COLLECTION_RECURSIVE
+  - !ruby/object:Api::Type::Enum
```

Comment:
is this field updatable? Is so, can we add/modify an update test to test it? See details at https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test

Reply:
No. Firestore does not support `index update` operation.